etca_asm/common_macros.py

```diff
@@ -8,14 +8,21 @@ def sign_extend(value, bits):
     return (value & (sign_bit - 1)) - (value & sign_bit)
 
 
-@common_macros.inst('/movx?/ register_raw "," immediate', strict=False)
-@common_macros.inst('/movx/ register_raw "," immediate', strict=True)
+@common_macros.inst('/mov/ register_raw "," immediate')
 def mov_large_immediate(context, _, reg, imm):
-    reject(not ((-0x8000 <= imm < -0x10) or (0x10 <= imm <= 0xFFFF)))
+    reject(not ((-0x8000 <= imm <= 0) or (0 <= imm <= 0xFFFF)))
```

Comment:
Does this do anything?

Reply:
Isn't `-0x8000 <= imm <= 0xFFFF` the same thing?

Reply:
Yes, that's better.